src/main/java/com/icthh/xm/uaa/security/DomainJwtAccessTokenDetailsPostProcessor.java

```diff
@@ -17,11 +17,11 @@
 
     @LogicExtensionPoint(value = "ProcessJwtAccessTokenDetails", resolver = OptionalProfileHeaderResolver.class)
     public void processJwtAccessTokenDetails(OAuth2Authentication authentication, Map<String, Object> details) {
-        Map<String, Object> additionalDetails = new HashMap<>();
-        additionalDetails.putAll(authentication.getOAuth2Request().getRequestParameters());
+        Map<String, Object> additionalDetails = new HashMap<>(authentication.getOAuth2Request().getRequestParameters());
         // password and client_secret removed by spring
         additionalDetails.remove("grant_type");
         additionalDetails.remove("username");
+        additionalDetails.remove("token");
```

Comment:
it's can broke existing tenant, that using token in additional deatils

Reply:
Remark applied: removed.